ui/src/app/dashboard/dashboard.component.spec.ts

```diff
@@ -67,9 +70,55 @@ describe('DashboardComponent', () => {
     expect(testComponent).toBeTruthy();
   }));
 
+  it("should create expected amount of cards", fakeAsync(() => {
+    let matCards = fixture.debugElement.queryAll(By.css('mat-card'));
+    expect(matCards.length).toEqual(CARD_NUM);
+  }));
+
+  it('should navigate to job-list page (fake) when status counts are clicked', fakeAsync(() => {
+    let countAnchor = fixture.debugElement.query(By.css("jm-dashboard a")).nativeElement;
+    countAnchor.click();
+    fixture.detectChanges();
+    tick();
+    expect(fixture.debugElement.query(By.css("div")).nativeElement.textContent).toEqual("fake job-list page");
+  }));
+
+  it('should has status as url param when totalSummaryComponent links are clicked', fakeAsync(() => {
+    let totalSummaryAnchor = fixture.debugElement.query(By.css("jm-total-summary .count a")).nativeElement;
```

Comment:
optional: Alias debugElement to make this test a bit less verbose: `const de = fixture.debugElement;`